.clang-tidy

```diff
@@ -16,6 +16,16 @@ misc-*,\
 modernize-*,\
 performance-*,\
 readability-*,\
+-readability-named-parameter,\
+-misc-unused-parameters,\
+-performance-unnecessary-value-param,\
+-hicpp-use-auto,\
+-modernize-use-auto,\
+-performance-move-const-arg,\
+-readability-container-size-empty,\
+-readability-implicit-bool-conversion,\
+-readability-inconsistent-declaration-parameter-name,\
+-readability-named-parameter,\
```

Comment:
`readability-named-parameter` excluded twice.

Reply:
How much time are we saving by disabling these checks?

Reply:
Build time - don't know, probably not a lot. Developer time - a significant amount since they are fairly likely to trip the dev making them do another CI run. Additionally, they are likely to combine with other checks, with only one being emittied. The net result being that you dutifully fix style issues only to see more an hour later after linting again.

Reply:
How does `readability-container-size-empty` 'combine with other checks'?